### DISCUSSION TITLE
Automatically deeply include self

### DISCUSSION THREAD

**Discussion by SupertigerDev** [prisma/prisma#17187] [Q&A]
how can i refer to the existing include? I want it to forever deeply nest. any way to do this?
```prisma
model Post {
  id          String   @id
  content     String


  commentToId  String?
  commentTo  Post? @relation(fields: [commentToId], references: [id], name: "commentTo", onDelete: SetNull)
  comments Post[] @relation(name: "commentTo")
}
```
```ts
export async function fetchPosts(opts: FetchPostsOpts) {

  const posts = await prisma.post.findMany({
    where: {
      createdById: opts.userId
    },
    orderBy: {createdAt: 'desc'},
    take: 50,
    include: {
      commentTo: {
        // how can i refer to the existing include? I want it to forever deeply nest. any way to do this?
      },
      ...include other stuff
    }
  });

  return posts;
}
```

EDIT: ah great, seems like its not supported by prisma!

**Comment by nurul3101** [comment]
For reference, here's a related feature request, if you could leave a 👍  to it, it would help our product team in prioritising it.

- https://github.com/prisma/prisma/issues/3725

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
